sem_covid/services/crawlers/scrapy_crawlers/spiders/eu_timeline_spider.py

```diff
@@ -13,7 +14,9 @@ class EUTimelineSpider(scrapy.Spider):
     url = 'https://ec.europa.eu/info/live-work-travel-eu/coronavirus-response/timeline-eu-action_en'
     presscorner_base_url = 'https://ec.europa.eu/commission/presscorner/detail'
 
-    def __init__(self, filename, *args, storage_adapter=None, **kwargs):
+    def __init__(self, *args, filename: str = config.EU_TIMELINE_JSON,
+                 storage_adapter=StoreRegistry().minio_object_store(config.EU_TIMELINE_BUCKET_NAME),
```

Comment:
it is not good to create an object because the methods are static
|
--> StoreRegistry().minio_object_store(config.EU_TIMELINE_BUCKET_NAME)